cli/module_loader.rs

```diff
@@ -134,30 +137,52 @@ impl CliModuleLoader {
     maybe_referrer: Option<ModuleSpecifier>,
   ) -> Result<ModuleSource, AnyError> {
     let code_source = if self.ps.npm_resolver.in_npm_package(specifier) {
-      let file_path = specifier.to_file_path().unwrap();
+      let is_cjs = self.ps.cjs_resolutions.lock().contains(specifier);
```

Comment:
Question: Why is `cjs_resolutions` a Mutex? Is it shared across threads? Assuming not, RefCell should be much cheaper.

Reply:
Yeah, the data in ProcState is shared across threads.